components/ProgressBar.vue

```diff
@@ -29,7 +29,7 @@ export default {
 	progress[value]::-webkit-progress-value {
 		background-color: var(--cl-primary-200);
 		min-width: 20px;
-		transition: all 750ms ease-in-out;
+		transition: width 750ms ease-in-out;
```

Comment:
Transitioning the width can be costful. Using `transform: scaleX(..)` and transitioning the transform would be a better solution